From fa7a562dda7302bf87230279064916e6dc2aa7e3 Mon Sep 17 00:00:00 2001
From: Ankur <ankurv@verma>
Date: Wed, 15 May 2024 20:01:54 -0700
Subject: [PATCH] fix_const

---
 dtl/Diff.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dtl/Diff.hpp b/dtl/Diff.hpp
index 8c9c774..57067f9 100644
--- a/dtl/Diff.hpp
+++ b/dtl/Diff.hpp
@@ -164,7 +164,7 @@ namespace dtl {
             return trivial;
         }
         
-        void enableTrivial () const {
+        void enableTrivial () {
             this->trivial = true;
         }
         
-- 
2.45.0

